examples/pipelines/burnout_on_video.pipe.in

```diff
@@ -0,0 +1,73 @@
+# ================================================================
+process input
+  :: video_input
+  video_filename   	= @EXAMPLE_DIR@/videos/drone-video.mp4
+  frame_time        	= .3333
+
+  video_reader:type	= vidl_ffmpeg
+
+# ================================================================
+process filter
+  ::image_filter
+  filter:type                    	= vxl_pixel_feature_extractor
+  filter:vxl_pixel_feature_extractor:enable_color = true # 3 channels
+  filter:vxl_pixel_feature_extractor:enable_gray = true # 1 channel
+  filter:vxl_pixel_feature_extractor:enable_color_commonality = true # 1 channel
+  filter:vxl_pixel_feature_extractor:enable_high_pass_box = true # 3 channels
+  filter:vxl_pixel_feature_extractor:enable_high_pass_bidir = true # 1 channel
+  filter:vxl_pixel_feature_extractor:enable_average = true # 3 channels
+  filter:vxl_pixel_feature_extractor:enable_aligned_edge = true # 3 channels
+  filter:vxl_pixel_feature_extractor:enable_convert = false # 3 channels
+
+  filter:vxl_pixel_feature_extractor:average:window_size = 25
+  filter:vxl_pixel_feature_extractor:average:type = window
+  filter:vxl_pixel_feature_extractor:average:output_variance = true
+
+  # TODO figure out what size kernel to used because this expects GDS
+  filter:vxl_pixel_feature_extractor:high_pass_box:kernel_width = 13
+  filter:vxl_pixel_feature_extractor:high_pass_box:kernel_height = 13
+  filter:vxl_pixel_feature_extractor:high_pass_box:treat_as_interlaced = false
+  filter:vxl_pixel_feature_extractor:high_pass_box:output_net_only = false
+
+  filter:vxl_pixel_feature_extractor:high_pass_bidir:kernel_width = 27
+  filter:vxl_pixel_feature_extractor:high_pass_bidir:kernel_height = 27
+  filter:vxl_pixel_feature_extractor:high_pass_bidir:treat_as_interlaced = false
+  filter:vxl_pixel_feature_extractor:high_pass_bidir:output_net_only = true
+
+  filter:vxl_pixel_feature_extractor:color_commonality:smooth_image = false
+  filter:vxl_pixel_feature_extractor:color_commonality:output_scale = 255
+
+# ================================================================
+process classifier
+  ::image_filter
+  filter:type                    	= vxl_hashed_image_classifier_filter
+  filter:vxl_hashed_image_classifier_filter:default_filename = @EXAMPLE_DIR@/models/default_burnout_600_iters
+
+# ================================================================
+process thresholder
+  ::image_filter
+  filter:type                    	= vxl_adaptive_threshold
```

Comment:
This doesn't seem to exist, I think you meant `vxl_threshold`?

Reply:
This is #1198. This also depends on #1206 and #1207. The integration branch, #1210, pulls everything together.